requirements.txt

```diff
@@ -1,4 +1,6 @@
 markdown>=2.6.8
+python.markdown.math>=0.2
```

Comment:
`python.markdown.math>=0.2` was removed from `requirements.txt` because it's not a required extension. If the user wants to use the extension, they should download the package themselves (this is detailed in the documentation).